src/test/kotlin/validation/AuthenticationValidatorTest.kt

```diff
@@ -0,0 +1,144 @@
+package se.uulm.snowballr.backend.validation
+
+import `in`.rcard.assertj.arrowcore.EitherAssert
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import se.uulm.snowballr.backend.assertInvalidResult
+import se.uulm.snowballr.backend.model.InvalidEmail
+import se.uulm.snowballr.backend.validation.AuthenticationValidator.validateRegisterRequest
+import snowballr.Authentication
+
+class AuthenticationValidatorTest {
+    @Nested
+    inner class RegisterRequest {
+        private val validRegisterRequestBuilder: Authentication.RegisterRequest.Builder =
+            Authentication.RegisterRequest
+                .newBuilder()
+                .setEmail("alice.smith@example.com")
+                .setFirstName("Alice")
+                .setLastName("Smith")
+                .setPassword("AAbb__00")
+
+        @Test
+        fun `When a valid request is validated, then no issue is returned`() {
+            val request = validRegisterRequestBuilder.build()
+            val result = validateRegisterRequest(request)
+
+            EitherAssert.assertThat(result).isRight()
+        }
+
+        @Test
+        fun `When an invalid email is validated, then the 'InvalidEmail' issue is returned`() {
+            val request =
+                validRegisterRequestBuilder
+                    .setEmail("invalid-email")
+                    .build()
+            val result = validateRegisterRequest(request)
+
+            assertInvalidResult(result, InvalidEmail::class.java)
+        }
+
+        @Test
+        fun `When a blank first name is validated, then the 'BlankField' issue is returned`() {
+            val request =
+                validRegisterRequestBuilder
+                    .setFirstName("")
+                    .build()
+            val result = validateRegisterRequest(request)
+
+            assertInvalidResult(result, se.uulm.snowballr.backend.model.BlankField::class.java)
```

Comment:
```suggestion
            assertInvalidResult(result, BlankField::class.java)
```
Same for the other assertions.